src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -91,13 +90,17 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
                 var session = sessionProvider.GetOrCreate(GuidList.InteractiveWindowRSessionGuid);
                 IReadOnlyList<IREvaluationResultInfo> infoList = null;
                 try {
-                    infoList = session.DescribeChildrenAsync(REnvironments.GlobalEnv, variableName, REvaluationResultProperties.HasChildrenProperty,
-                            RValueRepresentations.Str(), _maxResults).WaitTimeout(_maxWaitTime);
+                    infoList = session.DescribeChildrenAsync(REnvironments.GlobalEnv, 
+                               variableName, 
+                               REvaluationResultProperties.HasChildrenProperty | REvaluationResultProperties.AccessorKindProperty,
+                            RValueRepresentations.Str(null), _maxResults).WaitTimeout(_maxWaitTime);
```

Comment:
I meant passing `null` instead of `Str` (if we don't need the representation; I'm not sure if that is the case here or not). Otherwise, pass some reasonable value as an argument to `Str` to restrict the maximum length of representation.